templates/careers.mustache

```diff
@@ -39,6 +39,7 @@
           <h2>Get Started Now</h2>
           <p><strong>An Opportunity Center can help you get started on one of these career paths.</strong></p>
           <p>Sign up for the Opportunity Center job system and receive a personalized recommendation for next steps and a follow-up from a career advisor.</p>
+          <p><strong>Find information on certification and training programs <span class="link"><a href="https://careerpathnola.business/training-and-educucation" target="_blank">here</a></span>.</strong></p>
```

Comment:
You'll want to link something more descriptive than `here` for accessibility reasons.

You can reference the [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/WAI/WCAG20/quickref/) for widely accepted standards on interface accessibility.

For this particular issue, you'll want to read about [WCAG 2.0 Technique H30: Providing link text that describes the purpose of a link for anchor elements](https://www.w3.org/TR/WCAG20-TECHS/H30.html)